service/src/main/resources/HTMLTemplate.html

```diff
@@ -525,24 +525,28 @@ <h3 class="heading heading--two">Child
         case_data.children.firstChild is not empty and
```

Comment:
In children we have social worker telephone number, does this need to be confidential also?

Reply:
good Q, Abi says no. This just applies to details directly to do with the child

Reply:
Thanks for checking